Command/AbstractCommand.php

```diff
@@ -3,14 +3,14 @@
 namespace Sli\ExpanderBundle\Command;
 
 use Sli\ExpanderBundle\Misc\KernelProxy;
-use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
 /**
  * @author Sergei Lissovski <sergei.lissovski@gmail.com>
  */
-abstract class AbstractCommand extends ContainerAwareCommand
+abstract class AbstractCommand extends Command
```

Comment:
@cravler , what version of SF this Command class has been introduced in?

Reply:
from 2.0
https://github.com/symfony/symfony/blob/2.0/src/Symfony/Component/Console/Command/Command.php